assets/js/main.js

```diff
@@ -218,4 +218,14 @@ var dopplerUI = dopplerUI || {};
 
   });
 
-
+// Carousel
+$(document).ready(function () {
+  $(".dp-carousel-dot").on('click', function () {
+    var slideId = $(this).attr('data-select');
+    var carrousel = $(this).closest('.dp-carousel').attr('id');
+    $('#' + carrousel + ' .dp-carousel-slide').removeClass('active');
+    $('#' + carrousel + ' .dp-carousel-dot').removeClass('select');
```

Comment:
Why use a select here were the radio already has the checked property? 
The issue with the active is because a div has no property checked or selected, is different than radios.